File: c#/program12.cs
using System;

struct DigitX
{
    public int No;

    public DigitX(int no)
    {
        this.No = no;
    }

    public int CountFrequency()
    {
        int iDigit = 0, temp = this.No, count = 0;

        while(temp != 0)
        {
            iDigit = temp % 10;
            if(iDigit == 7)
            {
                count++;
            }
            temp = temp / 10;
        }
        return count;
    }
}

class program12
{
    public static void Main()
    {
        int value = 0;

        Console.WriteLine("Enter number : ");
        value = int.Parse(Console.ReadLine());

        DigitX obj = new DigitX(value);
        Console.WriteLine("Frequency of the 7 is : "+obj.CountFrequency());
        Console.WriteLine("Frequency of the 7 is : "+obj.CountFrequency());
    }
}

File: java/program12.java
import java.util.*;

class Digits
{
    public int CountFrequency(int iNo)
    {
        int iCount = 0;

        while(iNo != 0)
        {
            if((iNo % 10) == 7)
            {
                iCount++;
            }
            iNo = iNo / 10;
        }
        return iCount;
    }
}

class program12
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number");
        int iValue = sobj.nextInt();

        Digits obj = new Digits();
        
        System.out.println("Count of the 7 is : "+obj.CountFrequency(iValue));
    }
}